edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -161,8 +162,11 @@ def _check_and_cache_login_required_found(self, view_func):
         Checks for LoginRedirectIfUnauthenticated permission and caches the result.
         """
         view_class = _get_view_class(view_func)
-        view_permission_classes = getattr(view_class, 'permission_classes', tuple())
-        is_login_required_found = _includes_base_class(view_permission_classes, LoginRedirectIfUnauthenticated)
+        view_permissions = getattr(view_class, 'permission_classes', tuple())
```

Comment:
Nit: Not really introduced by you, but above we have:
```
view_permissions = list(getattr(view_class, 'permission_classes', []))
```
Is there a reason for the inconsistency?

Reply:
It looks like an older version of this code used to iterate over the permission_classes using `list.pop()`:

https://github.com/pwnage101/edx-drf-extensions/blob/1d470a29a16bcaf5c9247c8236def8a4be56b077/edx_rest_framework_extensions/auth/jwt/middleware.py#L56-L57

The recursive helper function _iter_included_base_classes appears to support any iterable, so it shouldn't matter whether a list or tuple is passed. That said, I'm happy to make it consistently a list.